src/api-client/withApollo.tsx

```diff
@@ -55,7 +55,8 @@ function createApolloClient() {
   });
 }
 
-export function getApolloClient() {
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export function getApolloClient(_context?: unknown) {
```

Comment:
Our gql codegen passes a `ctx` variable to `getApolloClient` despite us not using it

Reply:
TypeScript is structurally checked, so it's entirely fine for a function that takes no arguments to be passed where one that receives arguments is required:

```ts

type Checker = (...args: number[]) => boolean

// valid!
const nope: Checker = () => false
```

Reply:
That is to say you should probably remove the argument 😉

Reply:
love it!